hip/settings/base.py

```diff
@@ -152,6 +155,18 @@
     },
 ]
 
+# Azure auth using Python Social Auth
+# https://python-social-auth.readthedocs.io/en/latest/backends/azuread.html
+SOCIAL_AUTH_JSONFIELD_ENABLED = True
+SOCIAL_AUTH_AZUREAD_V2_TENANT_OAUTH2_KEY = os.environ.get("AZURE_CLIENT_ID")
```

Comment:
How does this work with the `V2` in the setting name? I don't see that setting documented anywhere, though I do see `SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_KEY`

Reply:
Excellent :eye: ! I didn't realize that it wasn't documented. I found it because I was poring through the code trying to figure out why the email address wasn't properly populating on the new account and then I saw this backend: https://github.com/python-social-auth/social-core/blob/master/social_core/backends/azuread_tenant.py#L112 Specifically, this piece of code made sure that the email address was updated in our DB: https://github.com/python-social-auth/social-core/blob/master/social_core/backends/azuread_tenant.py#L121-L123 (I'll add a comment to the settings here)

Reply:
Sounds good. Thanks!